src/integrationTest/java/uk/gov/hmcts/reform/sscs/service/EvidenceShareServiceIt.java

```diff
@@ -184,28 +197,26 @@ public void appealWithMrnDateOlderThan30Days_shouldGenerateDL16TemplateAndAndAdd
         assertEquals("filename1.pdf", documentCaptor.getValue().get(2).getName());
 
         verify(restTemplate).postForEntity(anyString(), any(), eq(byte[].class));
-        verify(evidenceManagementService).upload(any(),  eq("sscs"));
+        verify(evidenceManagementService).upload(any(), eq("sscs"));
         verify(ccdService).updateCase(any(), any(), any(), any(), eq("Uploaded dl16-12345656789.pdf into SSCS"), any());
         verify(bulkPrintService).sendToBulkPrint(any(), any());
 
         verify(ccdService).updateCase(any(), any(), eq(EventType.SENT_TO_DWP.getCcdType()), any(), eq(documentList), any());
     }
 
-    @Test(expected = NoMrnDetailsException.class)
+    @Test
     public void appealWithNoMrnDate_shouldNotGenerateTemplateOrAddToCcd() throws IOException {
```

Comment:
Change description

Reply:
@jackmaloney , I think the name of the method is still valid. This method is still checking a case with no MrnDate and ensuring it is not generating the template and adding to CCD.

Reply:
I renamed the method to capture we also update the case to flag the error.